Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -62,22 +62,30 @@ instance Obj.inhabited [Inhabited P.A] [Inhabited α] : Inhabited (P α) :=
 
 instance : Functor.{v, max u v} P.Obj where map := @map P
 
-protected theorem map_eq {α β : Type u} (f : α → β) (a : P.A) (g : P.B a → α) :
-    @Functor.map P.Obj _ _ _ f ⟨a, g⟩ = ⟨a, f ∘ g⟩ :=
+/-- We prefer `PFunctor.map` to `Functor.map` because it is universe-polymorphic. -/
+@[simp]
+theorem map_eq_map {α β : Type v} (f : α → β) (x : P α) : f <$> x = P.map f x :=
+  rfl
+
+@[simp]
+protected theorem map_eq (f : α → β) (a : P.A) (g : P.B a → α) :
+    P.map f ⟨a, g⟩ = ⟨a, f ∘ g⟩ :=
   rfl
 #align pfunctor.map_eq PFunctor.map_eq
 
-protected theorem id_map {α : Type v} : ∀ x : P α, id <$> x = id x := fun ⟨_a, _b⟩ => rfl
+@[simp]
+protected theorem id_map : ∀ x : P α, P.map id x = x := fun ⟨_, _⟩ => rfl
 #align pfunctor.id_map PFunctor.id_map
 
-protected theorem comp_map {α β γ : Type v} (f : α → β) (g : β → γ) :
-    ∀ x : P α, (g ∘ f) <$> x = g <$> f <$> x := fun ⟨_a, _b⟩ => rfl
-#align pfunctor.comp_map PFunctor.comp_map
+@[simp]
+protected theorem map_map (f : α → β) (g : β → γ) :
+    ∀ x : P α, P.map g (P.map f x) = P.map (g ∘ f) x := fun ⟨_, _⟩ => rfl
+#align pfunctor.comp_map PFunctor.map_map
```

Comment:
This is a fairly different lemma now, so
```suggestion
#align pfunctor.comp_map PFunctor.map_mapₓ
```

otherwise

bors d+

Reply:
`ₓ` is deprecated now.
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60.E2.82.93.60.20in.20.60.23align.60.20.235847